tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -91,25 +87,30 @@ protected function setUp(): void {
     $node->set('oe_documents', [$media_document]);
     $node->set('oe_featured_media', [$media_image]);
     $node->save();
-    $this->node = $node;
+    return $node;
   }
 
   /**
-   * Tests that the Event page renders correctly in full display.
+   * Tests the event page.
    */
-  public function testEventRenderingFull(): void {
-    // Build node full view.
-    $builder = \Drupal::entityTypeManager()->getViewBuilder('node');
-    $build = $builder->view($this->node, 'full');
-    $render = $this->container->get('renderer')->renderRoot($build);
-    $crawler = new Crawler((string) $render);
+  public function testEventPage(): void {
+    $node = $this->createExampleEvent();
+    $this->drupalGet('node/' . $node->id());
 
-    // Assert content banner title.
+    /** @var \Symfony\Component\BrowserKit\AbstractBrowser $client */
+    $client = $this->getSession()->getDriver()->getClient();
+    $crawler = $client->getCrawler();
+
+    // Select the content banner element.
     $content_banner = $crawler->filter('.bcl-content-banner');
+    $this->assertSame(1, $content_banner->count(), 'Content banner found.');
```

Comment:
```suggestion
    $this->assertSame(1, $content_banner->count(), 'Content banner not found.');
```
the message displays when this fails

Reply:
I found this in Drupal core a lot. But I now learn this is a relic from the past, and should go:
https://www.drupal.org/project/coding_standards/issues/2057905#comment-9470963
So I will change it.

Reply:
After talk with Francesco I am dropping the comment and use assertCount().